state/application_test.go

```diff
@@ -222,80 +222,6 @@ func (s *ApplicationSuite) TestSetCharmUpdatesBindings(c *gc.C) {
 	})
 }
 
-func (s *ApplicationSuite) TestSetCharmWithWeirdlyNamedEndpoints(c *gc.C) {
```

Comment:
I think this was the wrong place to test this anyway. The new tests in for the State charm methods seems like a better place for this.